### PR TITLE
configure and association changes

### PR DESCRIPTION
**Pull request by geekypandey**

### CODE REVIEW

File: src/app/pages/assess/assign/company-assessment/company-assessment-edit/CompanyAssessmentEditComponent.ts [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    for (const field of fields) {
+      this.editForm.controls[field].disable();
+    }
+  }
+
+  downloadCredits() {
+    const id = this.assessment.id;
+    this.assessmentService.downloadCredits(id).subscribe((value: any) => {
+      const blob = new Blob([value], { type: 'application/octect-stream' });
+      this._fileSaver.save(blob, 'Available_Credits.xlsx');
```
Comment (kiran-cm) [inline src/app/pages/assess/assign/company-assessment/company-assessment-edit/CompanyAssessmentEditComponent.ts:145]: we don't need fileSaver service.
I have created saveFile method in assess.utils.ts. Can you try with that

File: package.json [hunk 2]
```diff
@@ -28,6 +28,7 @@
     "merge-jsons-webpack-plugin": "2.0.1",
     "moment": "^2.30.1",
     "ngx-build-plus": "16.0.0",
+    "ngx-filesaver": "^17.0.0",
```
Comment (kiran-cm) [inline package.json:31]: Not required. use saveFile method added in assess.util.ts

File: src/app/pages/assess/assign/company-assessment/company-assessment-edit/CompanyAssessmentEditComponent.ts (hunk 1 above)
Comment (geekypandey) [reply src/app/pages/assess/assign/company-assessment/company-assessment-edit/CompanyAssessmentEditComponent.ts:145]: done

File: package.json (hunk 2 above)
Comment (geekypandey) [reply package.json]: 👍

**Review by kiran-cm — APPROVED** [review]

**Pull request merged**